src/main/java/org/apache/datasketches/kll/KllDoublesSketch.java

```diff
@@ -289,9 +334,20 @@ public double[] getQuantiles(final double[] fractions) {
    * A value of 2 will return the min and the max value. A value of 3 will return the min,
    * the median and the max value, etc.
    *
+   * @param inclusive if true, the fractional ranks are considered inclusive
    * @return array of approximations to the given fractions in the same order as given fractions
    * array.
    */
+  public double[] getQuantiles(final int numEvenlySpaced, final boolean inclusive) {
+    if (isEmpty()) { return null; }
+    return getQuantiles(org.apache.datasketches.Util.evenlySpaced(0.0, 1.0, numEvenlySpaced), inclusive);
+  }
+
+  /**
+   * Same as {@link #getQuantiles(int, boolean) getQuantiles(int numEvenlySpaced, false)}
+   * @param numEvenlySpaced number of evenly spaced fractional ranks
+   * @return quantiles
+   */
   public double[] getQuantiles(final int numEvenlySpaced) {
     if (isEmpty()) { return null; }
     return getQuantiles(org.apache.datasketches.Util.evenlySpaced(0.0, 1.0, numEvenlySpaced));
```

Comment:
Doesn't explicitly call getQuantiles with inclusive=false, although the effect is still the same. Shouldn't matter unless we break the API by changing the default behavior.